src/styles/_card.scss

```diff
@@ -0,0 +1,124 @@
+.container {
+  display: grid;
+  justify-content: center;
+
+  $grid-columns: 1;
+
+  grid-template-columns: repeat($grid-columns, 200px);
+  padding: 50px 40px;
+  gap: 46px 48px;
```

Comment:
Variables should be defined at the beginning of the file or in a separate file with variables. Move `$grid-columns` to the top of the file or to `_variables.scss` for better maintainability and readability.